examples/operations/QueueScript.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<p>Ok... so this is me just being annoying, but "good" and "bad" are probably not the proper adjective to use here. The configuration isn't "bad" in the fact that it's incomplete.</p><p> </p><p>Consider:</p><p>We will start with an intentionally invalid configuration that is missing a couple required fields.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>